chat-plugins/info.js

```diff
@@ -1655,7 +1655,7 @@ exports.commands = {
 		if (!target) return this.parse('/help htmlbox');
 		target = this.canHTML(target);
 		if (!target) return;
-		if (!this.can('declare', null, room)) return;
+		if (!this.can('declare', null, room) && this.broadcasting) return;
 		if (!this.runBroadcast('!htmlbox')) return;
```

Comment:
This is all backwards.

``this.broadcasting`` is always false until ``this.runBroadcast`` is called.

And unless you want to always display the bad permissions error, make sure to
``if (this.broadcasting && !this.can('declare', null, room)) return;``